### DISCUSSION TITLE
v3.1.1

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#736] [Announcements]
## Changes

- Add ethernet support for GL-inet GL-S10 @mjtimmer (#734)
- Generic SHTx sensor implemented @DTTerastar (#660)
- Bump loader-utils from 3.2.0 to 3.2.1 in /ui @dependabot (#729)
- Updated Issue Templates to Forms @Choromanski (#701)

<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense/releases/tag/v3.1.1'>v3.1.1</a>.</em>

**Comment by windskipper** [comment]
Hello, been recently using this release. How do you disable esp32 onboard LED ?

**Comment by gcortes** [comment]
I just reflashed all my ATOM esp32 devices and noticed that as well. It's not a problem, but would like to turn them off if possible. When I mistakenly flashed one with the standard version, the LED wasn't on.

While working on something else, I saw that there were settings for the LED's on the device configuration page. I tried setting the Pin to -1 to disable it and then save and reboot. The LED didn't go off. I then disconnected the device and plugged it back in. The light stayed off.
